tests/case_fieldmaps.py

```diff
@@ -0,0 +1,57 @@
+"""Fieldmap cases we want to test."""
+
+from mrinufft.extras import make_b0map, make_t2smap
+
+
+class CasesB0maps:
+    """B0 field maps cases we want to test.
+
+    Each case return a field map and the binary spatial support of the object.
+    """
+
+    def case_real2D(self, N=64, b0range=(-300, 300)):
+        """Create a real (B0 only) 2D field map."""
+        return make_b0map(2 * [N])
+
+    def case_real3D(self, N=64, b0range=(-300, 300)):
+        """Create a real (B0 only) 3D field map."""
+        return make_b0map(3 * [N])
+
+
+class CasesZmaps:
+    """Complex zmap field maps cases we want to test.
```

Comment:
Just a question, what do you mean by zmap here?

Reply:
I was following the notation from this paper from [Fessler and Noll](https://deepblue.lib.umich.edu/bitstream/handle/2027.42/85903/Fessler50.pdf?sequence=1&isAllowed=y). Happy to change it to something else if you think this is confusing!

Reply:
No I think I am okay with it, just wanted to know the reasoning